cfgmgr/buffermgr.h

```diff
@@ -11,7 +11,7 @@
 namespace swss {
 
 #define INGRESS_LOSSLESS_PG_POOL_NAME "ingress_lossless_pool"
-#define LOSSLESS_PGS "3-4"
+#define LOSSLESS_PGS "2,3-4,6"
```

Comment:
I don't think it's a good way to have 4 lossless PGs reserved for all ports. Can we reserve 4 lossless PGs for t0 uplink and t1 downlink in dual-ToR topo only?
I think there are 2 ways to do it
- reserve lossless PGs according to the roles of the ports which can be retrieved from `DEVICE_NEIGHBOR` table
- introduce an extra table or leverage the existing table to indicate the lossless PGs on a port

Reply:
the 'pfc_enable' field in PORT_QOS_MAP already captures the correct lossless PGs based on device role. We can use that to attach the lossless pgs in the BUFFER_PG table

Reply:
Updated. Thanks so much